index.js

```diff
@@ -1,64 +1,65 @@
-const Boom     = require('boom')
-const gimme    = require('@articulate/gimme')
-const jwks     = require('jwks-rsa')
-const jwt      = require('jsonwebtoken')
+const axios                 = require('axios')
+const deepmerge             = require('deepmerge')
```

Comment:
this module provides the same functionality and is 7kB vs 30kB.  I tested in run kit and it gives the same results for deepmerge's example.

https://www.npmjs.com/package/@ianwalter/merge

also I only saw one use of deepmerge and it seems like a basic spread would suffice

Reply:
👍 Smaller is good.